supportedBackends/htcondor/src/main/scala/cromwell/backend/impl/htcondor/HtCondorJobExecutionActor.scala

```diff
@@ -68,6 +71,30 @@ class HtCondorJobExecutionActor(override val jobDescriptor: BackendJobDescriptor
     HtCondorRuntimeAttributes(runtimeMap, jobDescriptor.descriptor.workflowOptions)
   }
 
+  private val cacheActor = context.actorOf(cacheActorProps, s"CacheActor-${jobDescriptor.call.fullyQualifiedName}")
+
+  log.debug("{} Calculating hash for current job.", tag)
+  private val jobHash = calculateHash
```

Comment:
you can move this line up before cacheActor, seems like this line may throw exception that would avoid creation of cacheActor.